x-pack/plugins/apm/public/components/app/settings/agent_explorer/agent_list/index.tsx

```diff
@@ -41,7 +40,7 @@ export function getAgentsColumns({
     {
       field: AgentExplorerFieldName.ServiceName,
       name: '',
-      width: `${unit * 3}px`,
+      width: '5%',
```

Comment:
in larger resolutions percentage might be odd.

Reply:
The problem lies in the fact that we should use `tableLayout=fixed` to be able to use `truncateText` functionality [(tables#table-layout)](https://eui.elastic.co/#/tabular-content/tables#table-layout), otherwise columns such as `version` will take a lot of space when they have bigger version names (even if it's just one).
If we don't use percentages we will be fixing all columns width, but we still want the `serviceName` to occupy as much as it's left not a fixed width; if we don't set a width we lose the column in intermediate resolutions as you pointed out in the issue.